pkg/registry/populator_test.go

```diff
@@ -868,3 +871,258 @@ func TestDeprecateBundle(t *testing.T) {
 		})
 	}
 }
+
+func TestOverwrite(t *testing.T) {
+	type args struct {
+		firstAdd   map[image.Reference]string
+		secondAdd  map[image.Reference]string
+		overwrites map[string]map[image.Reference]string
+	}
+	type pkgChannel map[string][]string
+	type expected struct {
+		err                      error
```

Comment:
unused. It seems we're missing negative cases; e.g. "I should get an error when overwriting a non-latest bundle"